tsconfig.json

```diff
@@ -9,7 +8,8 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "allowSyntheticDefaultImports": true,
-    "sourceMap": true,
+    "sourceMap": false,
+    "noEmit": false,
```

Comment:
Can we revert these changes if they weren't needed?

Reply:
I'm keeping this for now to minimise build size.